client/app/services/vm-details/vm-details.html

```diff
@@ -12,8 +12,20 @@
         </li>
       </ol>
   </span>
+  <div class="view-selector-bar">
+    <div class="form-group toolbar-pf-view-selector">
```

Comment:
I looked at this and these are just CSS classes not patternfly components.  I don't believe we need to do anything in this case.